srcpkgs/civetweb/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'civetweb'
+pkgname=civetweb
+version=1.15
+revision=1
+build_style=gnu-makefile
+make_install_target="install install-headers install-lib install-slib"
+make_use_env=yes
+hostmakedepends="openssl-devel"
+makedepends="openssl-devel"
+short_desc="Embedded C/C++ web server"
+maintainer="bingulo <gcarlos@disroot.org>"
+license="MIT"
+homepage="https://civetweb.github.io/civetweb/"
+distfiles="https://github.com/civetweb/civetweb/archive/refs/tags/v${version}.tar.gz"
+checksum=90a533422944ab327a4fbb9969f0845d0dba05354f9cacce3a5005fa59f593b9
+
+pre_install() {
+	vmkdir usr/include
+}
+
+post_install() {
+	vlicense LICENSE.md
+}
+
+civetweb-devel_package() {
+	depends="civetweb-${version}_${revision}"
+		short_desc+=' - development files'
+		pkg_install() {
+			vmove usr/include
+				vmove "usr/lib/*.a"
+				vmove "usr/lib/*.so"
+				vmove "usr/lib/*.so.1"
+		}
+}
+
+civetweb-doc_package() {
```

Comment:
I don't think there's enough documentation here to warrant a subpkg

```
/usr/share/doc/civetweb/CREDITS.md
/usr/share/doc/civetweb/LICENSE.md
/usr/share/doc/civetweb/README.md
/usr/share/doc/civetweb/RELEASE_NOTES.md
/usr/share/doc/civetweb/SECURITY.md
/usr/share/doc/civetweb/civetweb_64x64.png
/usr/share/doc/civetweb/index.html
```

Reply:
I'll remove the doc subpkg, thx.